recipe/meta.yaml

```diff
@@ -19,11 +19,11 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - make
+    - make            # [unix]
 
   host:
     - bzip2           # [unix]
-    - curl            # [unix]
+    - curl 7.61.1           # [unix]
```

Comment:
Is this still needed?

cc @marcelotrevisani

Reply:
Nope, it was just a test

Reply:
Given how long the CI is taking, and how low this is lying in the dependency stack, I'm tempted to leave this as-is ... but I can definitely take it out if folks would prefer.

Reply:
I also think that given that cmake is basically always a `build` time dependency things like exact consistency with the conda-forge curl matter less?